src/legacy/core_plugins/data/public/search/index.ts

```diff
@@ -16,3 +16,6 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+export * from './search_service';
+export * from './utils';
```

Comment:
Are we trying to get away from `export *`? Or does it not matter since it's in an index file?

Reply:
I definitely think we should get away from `export *` at the top level `public/index.ts` and `server/index.ts`. Internally I think it is less important since using it here won't leak anything into the public API, but I also have no reservations with just naming exports everywhere... will update.